.github/workflows/delete-preview.yml

```
@@ -0,0 +1,34 @@
+  pull_request:
+    types: [closed]
+    branches:
+      - main
+
+jobs:
+  delete-preview:
+    runs-on: x64
+    steps:
+      - name: Get branch name
+        id: get-branch
+        run: echo "branch=$(gh pr view $PR_NO --repo $REPO --json headRefName --jq '.headRefName')" >> $GITHUB_OUTPUT
```

actionlint

shellcheck reported issue in this script: SC2086:info:1:27: Double quote to prevent globbing and word splitting

(shellcheck)